cdk/apps/rnasum/stacks/cicd.py

```diff
@@ -1,46 +1,52 @@
 from aws_cdk import (
     core,
     aws_codebuild as cb,
-    aws_ecr as ecr,
     aws_iam as iam,
     aws_s3 as s3,
 )
 
+# As semver dictates: https://regex101.com/r/Ly7O1x/3/
+semver_tag_regex = '(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$'
+
+
 class CICDStack(core.Stack):
-    def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
+    def __init__(self, scope: core.Construct, id: str, props, **kwargs) -> None:
         super().__init__(scope, id, **kwargs)
 
-        # As semver dictates: https://regex101.com/r/Ly7O1x/3/
-        semver_tag_regex = '(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$'
-
-        refdata = s3.Bucket.from_bucket_attributes(
-            self, 'reference_data',
-            bucket_name='umccr-refdata-dev'
+        # RNAsum doesn't need refdata at the moment
```

Comment:
I thought it does...

Reply:
I mean, when testing/QA-verifying RNAsum after image has built. Anyway, just turned it on back. I reckon, it will get it use in future. Something to improve on RNAsum dev side.